core/src/main/java/org/mskcc/cbio/portal/scripts/NormalizeExpressionLevels.java

```diff
@@ -218,10 +225,11 @@ private static void computeZScoreXP(String file){
             values = line.split("\t");
                     
             CanonicalGene gene;
-            if (values[1].matches("[0-9]+")) {
-                gene = daoGeneOptimized.getGene(Long.parseLong(values[1]));
-            } else {
-                gene = daoGeneOptimized.getNonAmbiguousGene(values[0]);
+            if (entrezGeneIdIndex != -1) {
```

Comment:
here we prefer to use entrezGeneId if the column is available (over hugo gene symbol).
We could consider using hugo gene symbol if a record has a blank in entrezGeneId ... but we probably expect all entrezGeneId values to be populated if the column is present.

Reply:
This behavior is consistent in other places of the codebase, like [ImportTabDelimData.java](https://github.com/cBioPortal/cbioportal/blob/master/core/src/main/java/org/mskcc/cbio/portal/scripts/ImportTabDelimData.java).